### ISSUE TITLE
Inference problem

### ISSUE THREAD

**Issue by yangzhifeio** [chenguolin/InstructScene#8]
Thanks for your great work.
There is no texture in the scene diagram of topdown perspective generated by me when I reproduce it. I would like to ask what the specific reason is.
Looking forward to your reply.
Thanks.

**Comment by chenguolin** [comment]
That may be caused by a problem with the file path of the texture image. You can check the filename of the texture image in the `model.mtl` of the 3D-FRONT dataset.

**Comment by AkiiLucky** [comment]
I have the same problem. No texture in the scene diagram.

![Image](https://github.com/user-attachments/assets/770d74ef-07e2-4e42-b293-edd6d0af0db0)

**Comment by sculmh** [comment]
@AkiiLucky  It seems that the texture file name in the MTL is incorrect. 

The author's code generates a temporary tmp_dir containing three types of files: OBJ, MTL, and texture files, with names like object_00.obj, object_00.png, and object_00.mtl. 
I found that the map_Kd in the MTL, which refers to the texture file name, points to a non-existent image, such as solid_001_wire.png. When I used Blender to generate a GLB for visualization from the files in tmp_dir, it appeared as a white model without texture materials. After I modified the map_Kd in the MTL to material_{number}.png and then combined them into a GLB for visualization, the textures appeared. 

<img width="186" height="382" alt="Image" src="https://github.com/user-attachments/assets/61bc3693-7d02-4f8b-be3c-8d5f5b909723" />

<img width="401" height="188" alt="Image" src="https://github.com/user-attachments/assets/ecb780e3-1b07-414f-94dd-01bb8c1d3b92" />


Therefore, I believe there might be a bug causing the incorrect texture path reference in the intermediate MTL file.
